modules/indexing/src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/query/PatchQuery.scala

```diff
@@ -24,13 +24,11 @@ object PatchQuery {
     val subj     = NodeFactory.createURI(id.qualifyAsString)
     val graphUri = NodeFactory.createURI(graph.toString())
     val (construct, where) = predicates.zipWithIndex
-      .foldLeft((new ConstructBuilder, new ConstructBuilder)) {
+      .foldLeft((new ConstructBuilder, new SelectBuilder)) {
```

Comment:
what's the reason for the changes throughout the file from ConstructBuilder to SelectBuilder?

Reply:
Because of Jena 3.3